src/JoomlaBrowser.php

```diff
@@ -1099,26 +1042,195 @@ public function createCategory($title, $extension = '')
 		{
 			$extension = '&extension=' . $extension;
 		}
-
 		$this->amOnPage('administrator/index.php?option=com_categories' . $extension);
-
 		$this->waitForElement(['class' => 'page-title']);
 		$this->checkForPhpNoticesOrWarnings();
-
 		$this->debug('Click new category button');
 		$this->click($this->locator->adminToolbarButtonNew);
-
 		$this->waitForElement(['class' => 'page-title']);
-
 		$this->checkForPhpNoticesOrWarnings();
 		$this->fillField(['id' => 'jform_title'], $title);
-
 		$this->debug('Click new category apply button');
 		$this->click($this->locator->adminToolbarButtonApply);
-
 		$this->debug('see a success message after saving the category');
-
 		$this->see('Category saved', ['id' => 'system-message-container']);
 		$this->checkForPhpNoticesOrWarnings();
 	}
+
+	/**
+	 * Site Offline
+	 *
+	 * @return  void
+	 *
+	 * @since   3.7.5
+	 */
+	public function setSiteOffline($enable == true)
+	{
+		$this->debug('I set the site to offline');
+		$this->doAdministratorLogin();
+		$this->amOnPage('/administrator/index.php?option=com_config');
+		if($enable == true){
+			$this->click(['id' => 'jform_offline1']);
+		}else{
+			$this->click(['id' => 'jform_offline0']);
+		}
+		$this->click("Save");
+		$this->see('Configuration saved', ['id' => 'system-message-container']);
+	}
+
+	/**
+	 * Search Engine Optimization
+	 *
+	 * @param   \AcceptanceTester   $I  Acceptance Tester
+	 *
+	 * @return void
+	 */
+	public function setSiteSearchEngineFriendly($enable == true)
+	{
+		$this->debug('I set the Search engine optimisation to Yes');
+		$this->doAdministratorLogin();
+		$this->amOnPage('/administrator/index.php?option=com_config');
+		// If-else statement
+		if ($enable == true){
+			$this->click(['id' => 'jform_sef1']);
+			//Select Option 'After'
+			$this->selectOption(['id' => 'jform_sitename_pagetitles'],'After');
+		}
+		else
+		{
+			$this->click(['id' => 'jform_sef0']);
+		}
+		$this->click("Save");
+		$this->see('Configuration saved', ['id' => 'system-message-container']);
+	}
+
+	/**
```

Comment:
@Samgithub1997  have you checked style here? I don't think it is according to the way we discussed.